client/my-sites/plans/jetpack-plans/product-grid/index.tsx

```diff
@@ -282,6 +271,17 @@ const ProductGrid: React.FC< ProductsGridProps > = ( {
 				</div>
 			</ProductGridSection>
 			<StoreFooter />
+			<ul className="product-grid__asterisk-list">
+				<li className="product-grid__asterisk-item">
+					{ translate( '* Monthly plans are 7-day money back guarantee.' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Monthly plans are 7-day money back guarantee.' )` *ES Score: 11*